### PR TITLE
Add v1.13.1 release notes and align CI to other release branches

### PR DESCRIPTION
**Pull request by lnovara**
As per PR title.

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -32,8 +32,10 @@ steps:
       VALIDATE_KUBERNETES_KUBEVAL: "false"
       # We use a lot of copy paste in our code, is intended. So it important to don't use this linter
       VALIDATE_JSCPD: "false"
-      VALIDATE_YAML: "false"
```
Comment (ralgozino) [inline .drone.yml:35]: watch out here, we would be deleting again the YAML validation

**Review by nutellinoit — APPROVED** [review]
LGTM

**Review by ralgozino — CHANGES_REQUESTED** [review]

File: .drone.yml (hunk 1 above)
Comment (lnovara) [reply .drone.yml:35]: No, by removing that line I am enabling it.

Comment (lnovara) [reply .drone.yml:35]: Indeed, if you look at the CI logs, you'll see some warnings about YAML formatting.

Comment (ralgozino) [reply .drone.yml:35]: lol, you are right 🤦 sorry

**Review by ralgozino — APPROVED** [review]

**Pull request merged**